base_layer/p2p/src/peer_seeds.rs

```diff
@@ -199,7 +199,6 @@ mod test {
         use super::*;
         use crate::{dns::mock, DEFAULT_DNS_NAME_SERVER};
 
-        #[ignore = "This test requires network IO and is mostly useful during development"]
```

Comment:
This test requires the DNS to be available which is why I ignored it. But we can keep it in and if `seeds.weatherwax.tari.com` ever goes away we'll fix it then.